### DISCUSSION TITLE
Is Shohei Ohtani Swinging More Strikes on Low Pitches This Season?

### DISCUSSION THREAD

**Discussion by StuffbyYuki** [has2k1/plotnine#837] [2024 Plotnine Contest]
### Authors

Yuki Kakegawa

### Links

[linkedin](https://www.linkedin.com/in/yukikakegawa/), [consulting company](https://www.oremdata.com/), [personal website](https://yukikakegawa.me/), [medium](https://medium.com/@yukithejapanese)

### Full description

- This analysis is for me to look at the data to confirm my assumption/observations that Ohtani is getting fewer swinging strikes on low pitches this 2024 season. And that might be one of the reasons why Ohtani is hitting much better than the last few seasons. 
- We create scatter plots showing pitches thrown to Ohtani, categorized by pitch locations and whether he swung or not.  
- A table visual with great_tables showing metrics in a table.

### Code repository

https://github.com/StuffbyYuki/Plotnine-Contest/blob/main/2024/shohei_ohtani_analyses/batting_swinging_strike_analysis.ipynb

**Comment by has2k1** [comment]
![shohei-ohtani-swings](https://github.com/user-attachments/assets/1c273eb5-94e3-4d16-8678-3b088eb1e66d)

**Comment by has2k1** [comment]
@StuffbyYuki, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you again for taking part in the contest.
